src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -255,11 +285,11 @@ public void runVelocity(ChassisSpeeds speeds) {
   }
 
   /** Runs the drive in a straight line with the specified drive output. */
-  public void runCharacterization(double output) {
-    for (int i = 0; i < 4; i++) {
-      modules[i].runCharacterization(output);
-    }
-  }
+  // public void runCharacterization(double output) {
+  //   for (int i = 0; i < 4; i++) {
+  //     modules[i].runCharacterization(output);
+  //   }
+  // }
```

Comment:
It looks like we just moved this function, but left it commented in the original location?